roles/matrix-postgres-backup/defaults/main.yml

```diff
@@ -0,0 +1,140 @@
+matrix_postgres_backaup_enabled: false
+
+matrix_postgres_connection_hostname: "matrix-postgres"
+matrix_postgres_connection_port: 5432
+matrix_postgres_connection_username: "matrix"
+matrix_postgres_connection_password: ""
+
+matrix_postgres_backup_extra_opts: "-Z9 --schema=public --blobs"
+matrix_postgres_backup_schedule: "@daily"
+matrix_postgres_backup_keep_days: "7"
+matrix_postgres_backup_keep_weeks: "4"
+matrix_postgres_backup_keep_months: "12"
+matrix_postgres_backup_healthcheck_port: "8080"
+matrix_postgres_backup_db_list: ""
+matrix_postgres_backup_path: "{{ matrix_base_data_path }}/postgres-backup"
+
+matrix_postgres_base_path: "{{ matrix_base_data_path }}/postgres"
+matrix_postgres_data_path: "{{ matrix_postgres_base_path }}/data"
+
+# uses sqlite per default
```

Comment:
Instead of defining all of these enabled variables and `_database_name` variables and `matrix_postgres_backup_db_dict`, we may be able to redo this in a way that automatically enables backups for all of our databases.

Take a look at `matrix_postgres_additional_databases` (especially its definition in `group_vars/matrix_servers`).

You can forward that whole variable to `matrix_postgres_backup_databases` via `group_vars/matrix_servers` too, and then loop over it and back things up?

If we'd like to allow for more flexibility (people saying "don't back up this specific database" in the future), we may add some additional exclusion list feature (e.g. `matrix_postgres_backup_database_names_to_exclude: []`). But that can be added later, I think.

Reply:
Great idea. I will rework this.

Reply:
I reworked this to use the matrix_postgres_additional_databases variable in group_vars/matrix_servers